src/pages/dar_application/DataUseAgreements.js

```diff
@@ -0,0 +1,105 @@
+import { a, div, fieldset, h, h3, label, li, ol, p, span} from 'react-hyperscript-helpers';
+import isNil from 'lodash/fp/isNil';
+import { Alert } from '../../components/Alert';
+import ReactTooltip from 'react-tooltip';
+
+export default function DataUseAgreements(props) {
+
+  const {
+    darCode,
+    showValidationMessages,
+    problemSavingRequest,
+    attestAndSave,
+    ConfirmationDialogComponent,
+    partialSave,
+    prevPage
+  } = props;
+
+  return (
+    div({ className: 'col-lg-10 col-lg-offset-1 col-md-12 col-sm-12 col-xs-12' }, [
```

Comment:
Just TOL (thinking out loud) ... would love to get rid of these bootstrap column styles some day. I don't have a clean technical approach to doing this yet, but will keep ruminating on it.